src/pages/Home/Home.js

```diff
@@ -1,25 +1,125 @@
-import { Button } from '../../components';
+import { Button, GameTypes, RulesTextBox } from '../../components';
 import { html } from '../../shared';
+import rickAndMorty from '../../public/img/HomeRickAndMorty.png';
+import styles from './Home.module.css';
 
 /**
  * @param {{} & import('..').RouterProps} props
  * @returns
  */
+
 function Home({ router }) {
-  const pages = [1, 2, 3, 4, 5];
-  return html`<div class="vstack">
-    <h2>Home</h2>
-    ${pages.map((x) =>
-      Button({
-        onClick: (e) => {
-          e.preventDefault();
-          router.goto({ page: 'todo', data: { id: x } });
-        },
-        text: `Go to page ${x}`,
-        variant: 'normal',
-      }),
-    )}
-  </div>`;
+  let selectedCategory = null;
+  let selectedDifficulty = null;
+
+  const rulesBox = RulesTextBox(
+    'Opis poziomu',
+    'What is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing and',
+  );
```

Comment:
Wydaje mi się, że skoro i tak mamy na początku `display: none`, to można zrobić po prostu tak:
```suggestion
 const rulesBox = RulesTextBox('', '');
```